apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -284,43 +284,41 @@ const CustomizableCurriculumCatalogCard = ({
               )}
             >
               <Button
-                color={Button.ButtonColor.neutralDark}
-                type="button"
                 onClick={onQuickViewClick}
-                aria-label={quickViewButtonDescription}
+                ariaLabel={quickViewButtonDescription}
                 text={i18n.quickView()}
                 className={`${style.buttonFlex} ${style.quickViewButton}`}
+                type="secondary"
+                color="black"
```

Comment:
I left a similar comment on one of Turner's PRs, but Erin Bond made a good case (https://github.com/code-dot-org/code-dot-org/pull/59924#discussion_r1687241002) for using the `buttonColors` constant on this component.